components/agenda/src/index.html

```diff
@@ -13,7 +13,10 @@
         const titleInput = document.querySelector("input#title");
         const startTimeInput = document.querySelector("input#start-time");
         const endTimeInput = document.querySelector("input#end-time");
-
+        agenda.id =
```

Comment:
Would it make more sense to have cypress overwrite the components' ID prop in a `beforeEach()`?

Reply:
I second this if possible; has better separation of concerns.

Reply:
that does make more sense. I'll do that. Thanks, @ozsivanov